server/test/db/timeslotModel.test.js

```diff
@@ -0,0 +1,57 @@
+const timeslotModel = require('../../models/db/timeslot');
+const moment= require('moment');
+
+describe('testing parameters for insertWorkerTimeslot method', () => {
+    beforeEach(() => {
+        jest.resetModules(); // Clears any cache between tests.
+    });
+
+    test('null value for all parameters', async () => {
+        // Arrange
+        const workerTimeslotId = null;
+        const slotId = null;
+        const schoolId = null;
+        const workerId = null;
+        const status = null;
+        const date = null;
+
+        // Act
+        const isTimelslotInserted = await timeslotModel.insertWorkerTimeslot(workerTimeslotId, slotId, schoolId, workerId, status, date);
```

Comment:
Hi Peter,
I thought that the insertWorkerTimeslot method only takes in slotId, schoolId, workerId, and date. I would recommend updating your tests to reflect the newest changes to the methods and running `npm test` in the server folder to ensure all tests pass before making a PR.

Reply:
Yes, it seems like I got ahead of myself. Will update the test code and run npm test to confirm tests pass.